cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AliasesTest.scala

```diff
@@ -192,20 +209,69 @@ class AliasesTest extends DocumentingTest with QueryStatisticsTestSupport {
           resultTable()
         }
+        p("Just as the local aliases, the remote database aliases can be given properties.")
+        query(
+          """CREATE ALIAS `remote-northwind-2021`
+            |FOR DATABASE `northwind-graph-2021` AT 'neo4j+s://location:7687'
+            |USER alice PASSWORD 'password'
+            |PROPERTIES { newestNorthwind: false, index: 6 }""".stripMargin,
+          ResultAssertions(r => { assertStats(r, systemUpdates = 1) })
+        ) { statsOnlyResultTable() }
+        p("The properties are then shown in the `SHOW ALIASES FOR DATABASE YIELD ...` command.")
+        query("SHOW ALIAS `remote-northwind-2021` FOR DATABASE YIELD name, properties", assertNameField("remote-northwind-2021")) { resultTable() }
         p(
           """Creating remote aliases also allows `IF NOT EXISTS` and `OR REPLACE` clauses.
             |Both check for any remote or local database aliases.""".stripMargin)
       }
+
+      section(title = "Create database aliases in composite databases", id = "database-management-create-database-alias-in-composite", role = "enterprise-edition") {
+        p(
+          """Both local and remote database aliases can be part of <<cypher-manual#administration-databases-create-composite-database, composite databases>>.
+            |Creating a database alias in a composite database is done by giving the composite database name as namespace for the alias name.""".stripMargin)
+        query(
+          """CREATE ALIAS garden.flowers
+            |FOR DATABASE `perennial-flowers`""".stripMargin,
+          ResultAssertions(r => { assertStats(r, systemUpdates = 1) })
+        ) { statsOnlyResultTable() }
+        query(
+          """CREATE ALIAS garden.trees
+            |FOR DATABASE trees AT 'neo4j+s://location:7687'
+            |USER alice PASSWORD 'password'""".stripMargin,
+          ResultAssertions(r => { assertStats(r, systemUpdates = 1) })
+        ) { statsOnlyResultTable() }
+        p("When a database alias has been created in a composite database, it will show up in the `constituents` column provided by the command `SHOW DATABASES` and in the `SHOW ALIASES FOR DATABASE` command.")
+        query("SHOW DATABASE garden YIELD name, constituents", assertNameField("garden")) { resultTable() }
+        query("SHOW ALIASES FOR DATABASE WHERE name STARTS WITH 'garden'", assertNameField("garden.flowers", "garden.trees")) { resultTable() }
+        note {
+          p("Aliases cannot point to a composite database.")
+        }
+        // Query fails with `key not found: org.neo4j.cypher.docgen.tooling.ErrorOnlyTablePlaceHolder@4f1502d7`
+        // if it is placed inside the note :(
+        query("CREATE ALIAS yard FOR DATABASE garden", ErrorAssertions(error =>
+          error.getMessage should startWith("Failed to create the specified database alias 'yard': Database 'garden' is composite.")
```

Comment:
I agree that it probably should be, but I also see the same thing for the error example in [TERMINATE TRANSACTIONS](https://development.neo4j.dev/docs/cypher-manual/5.0/clauses/transaction-clauses/#query-terminate-transactions) so I think it's something about the `errorOnlyResultTable` that is broken :(

Reply:
Adding to this, it might also be nice to not have the `run in browser` button on failing examples

Reply:
Maybe @recrwplay can help with that?

Reply:
For the erroronlyResultTable output it looks like we were missing a css class for that output. I've created https://github.com/neo4j-documentation/docs-ui/pull/119 to use the same style as resultsonly.

For the code block if you can get `role=noheader` into the asciidoc that is generated, so that it looks like this:

```
.Query
[source, cypher, role=noheader]
----
CREATE ALIAS yard FOR DATABASE garden
----
```

That will remove the entire header section. To just remove the 'Run in Neo4j Browser` button, you can use `role=noplay`. I don't know how that would be specified in the source, though. I'm not sure if the tests allow for the options for a codeblock to be changed in that way.

Reply:
I think I've managed to add the `role=noplay` on the error examples now :)